pom.xml

```diff
@@ -773,17 +804,6 @@ limitations under the License.
                 </plugins>
             </build>
         </profile>
-        <profile>
```

Comment:
As we're testing Java 17 in nightly build, shall we create a `java-17-target` profile and activate it there?

Reply:
Done.